Directory.Build.props

```diff
@@ -29,7 +29,7 @@
         continue to target .NET 7, so bump a 6 here to 7.
      -->
     <LatestDotNetCoreForMSBuild>$(NetCurrent)</LatestDotNetCoreForMSBuild>
-    <LatestDotNetCoreForMSBuild Condition=" '$(NetCurrent)' == 'net6.0' ">net7.0</LatestDotNetCoreForMSBuild>
+    <LatestDotNetCoreForMSBuild Condition=" '$(NetCurrent)' == 'net6.0' ">net8.0</LatestDotNetCoreForMSBuild>
```

Comment:
This should be deleted. NetCurrent in Arcade v8 is net8.0 already.

```suggestion
```

Reply:
+ the 2 comment lines above it should go with it